dali/ft/filecopy.cpp

```diff
@@ -3284,42 +3280,42 @@ void FileSprayer::updateTargetProperties()
             newRecord->addProp("@timestamp",temp.getString(timestr).str());
 
             StringBuffer val;
-            srcAttr->getProp("@owner", val);
-            newRecord->addProp("@owner", val.str());
+            //srcAttr->getProp("@owner", val);
+            newRecord->addProp("@owner", srcAttr->queryProp("@owner"));
 
-            srcAttr->getProp("@workunit", val.clear());
-            if (0 == val.length())
-                val.set("N/A");
-            newRecord->addProp("@workunit", val.str());
+            if (srcAttr->hasProp("@workunit"))
+                newRecord->addProp("@workunit", val.str());
```

Comment:
btw addProp works for attributes (@) but setProp is more correct, since you can't have multiple attributes of the same name.